vscode-trace-webviews/src/trace-viewer/vscode-trace-viewer-container.tsx

```diff
@@ -62,6 +60,12 @@ class TraceViewerContainer extends React.Component<{}, VscodeAppState>  {
       }
   };
 
+  private markerCategoriesMap: Map<string, string[]> = new Map<string, string[]>();
+  private toolbarMarkerCategoriesMap: Map<string, { categoryCount: number, toggleInd: boolean }> = new Map();
+  private selectedMarkerCategoriesMap: Map<string, string[]> = new Map<string, string[]>();
+  private markerSetsMap: Map<MarkerSet, boolean> = new Map<MarkerSet, boolean>();
```

Comment:
In general, objects don't make very good keys for Maps. If this is a map from marker -> enablement, then maybe the structure should be:

```typescript
// Key === MarkerSet.id
Map<string, {marker: MarkerSet, enabled: boolean}>
```

Then you would get the easy retrievability that Maps are designed for rather than having to cycle through `.keys()`.